src/__tests__/BrahmosPureComponent.test.js

```diff
@@ -0,0 +1,70 @@
+/*
+Inspired by -
+https://github.com/facebook/react/blob/master/packages/react/src/__tests__/ReactPureComponent-test.js
+*/
+import Brahmos, { render } from "..";
+
+describe("BrahmosPureComponent", () => {
+  it("should render", () => {
```

Comment:
Change test case statement.
`should rerender only when old props and new props are not shallow equal.`